lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -13,6 +13,11 @@ def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options
         DOTIW::Methods.distance_of_time_in_words(from_time, to_time, include_seconds_or_options, options)
       end
 
+      def distance_of_time_in_words_to_now(to_time = 0, include_seconds_or_options = {}, options = {})
+        return _distance_of_time_in_words(Time.now, to_time, options) if options.delete(:vague)
```

Comment:
Dup options or just `.key?(:vague)` if that's acceptable, see https://code.dblock.org/2020/11/11/dup-dont-modify-input-hash-parameters-in-ruby.html

Reply:
I was just following the pattern laid down in the method above it: https://github.com/radar/distance_of_time_in_words/blob/master/lib/dotiw/action_view/helpers/date_helper.rb#L12

Reply:
Yes, that's a bug. I didn't get to fix it yet, feel free to jump in!

https://github.com/radar/distance_of_time_in_words/issues/113

Reply:
Nice, two birds, one stone. Do you mind if I change both in this PR or do you need a separate PR for that other method?

Reply:
No need for a separate PR, but when I was refactoring a bunch of stuff I had tried to fix it and it didn't just work, so I opened an issue. So you might get stuck ;)

Reply:
Yup, you're right! Just tried it and got some failing tests. Let's get this one merged in and we'll deal with that one in separate PR. I've updated the CHANGELOG now and I'm using `.key?` instead of `.delete`. Take another look and let me know if there's anything else.

Reply:
Wait, hang on. I realize why `.key?` won't work out of the box. Because if you pass `vague: false` then it should use dotiw's detailed duration instead of the original method. Hang on a sec.

Reply:
I see. So when you don't delete the `vague` key, this error gets raised: 

```
ArgumentError:
       Unknown key: :vague. Valid keys are: :words_connector, :two_words_connector, :last_word_connector, :locale
     # ./lib/dotiw/methods.rb:96:in `_display_time_in_words'
     # ./lib/dotiw/methods.rb:29:in `distance_of_time_in_words'
     # ./lib/dotiw/action_view/helpers/date_helper.rb:20:in `distance_of_time_in_words_to_now'
     # ./spec/lib/dotiw_spec.rb:374:in `block (7 levels) in <top (required)>'
```

So, in this case, it seems like deleting `vague` is actually what we want. Passing `vague` bypasses the `dotiw` library and just uses the default method. There is no other use for it down the line in `dotiw` and, moreover, `dotiw` doesn't like receiving it as a `key`. 

Thoughts?

Reply:
Makes sense. Make sure to `options = options.dup`, then `options.delete(:vague)`. If this is always with Rails, then we can do `options.except(:vague)`.

Reply:
Good call on `except`, that worked great.

Take a look now and see what you think...

Reply:
> Yes, that's a bug. I didn't get to fix it yet, feel free to jump in!
> 
> #113

Ended up doing this in a separate PR: 

https://github.com/radar/distance_of_time_in_words/pull/119

Didn't run into any issues with dup'ing the `options`.